### PR TITLE
[DOCS-6823] Update New Folder, Move, Rename, Scan

### PR DESCRIPTION
**Pull request by Amy-Hyland**
This is a first draft of the features in Mobile Workspace: New Folder, Move files. Rename files, and Scan files. Additional information requested has been added to the ticket.

### CODE REVIEW

File: mobile-workspace/latest/using/index.md [hunk 1]
```diff
@@ -63,4 +62,73 @@ This will upload files to the Personal files area. If you want to upload to a sp
...
+
+## Create New Folder
+
+You can create new folders within the Mobile Workspace.
+
+1. Log into the Mobile Workspace.
+
+2. Search or navigate to the location you want to create a folder.
+
+3. Press the plus **+** button.
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:75]: Is the plus (+) button the same as the floating action button?

Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:75]: Is the plus (+) button the same as the "floating action" button?

```suggestion
3. Press the floating action button (**+**) button.
```

File: mobile-workspace/latest/using/index.md [hunk 2]
```diff
@@ -63,4 +62,73 @@ This will upload files to the Personal files area. If you want to upload to a sp
...
+
+## Move Files
+
+You can move files within the Mobile Workspace.
+
+1. Log into the Mobile Workspace.
+
+2. Search or navigate to the file you want to move.
+
+3. Select the Floating action button.
```
Comment (anxumalo) [inline mobile-workspace/latest/using/index.md:91]: See comment for line 75 - suggested usage:
```suggestion
3. Select the floating action button (+).
```

**Review by anxumalo — CHANGES_REQUESTED** [review]
Please see my inline comments

File: mobile-workspace/latest/using/index.md [hunk 3]
```diff
@@ -63,4 +62,73 @@ This will upload files to the Personal files area. If you want to upload to a sp
...
+4. Press **Scan documents**.
+* Your device's camera will load.
+* On the first use, you will have to allow the Mobile Workspace access to use the camera on your device.
+
+5. Position your device to scan and capture a photo of the document. Repeat for additional pages or documents to scan as one file.
+
+6. When all documents are scanned, select **Save**.
+
+* The scan will be saved where you navigated.
+* If you do not wish to keep the scan, exit without saving. A dialog box will appear to confirm you want to **Discard** the scan. Select **Cancel** to return to save your scan.
```
[the remark this answers is not in the document]
Comment (Amy-Hyland) [reply mobile-workspace/latest/using/index.md]: Updated in my latest commit

**Review by Amy-Hyland — COMMENTED** [review]
I've incorporated all the changes in my latest commit.

**Review by anxumalo — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**